ramls/examples/inventory-items-and-holdings.json

```diff
@@ -0,0 +1,48 @@
+{
+  "instanceId": "1b74ab75-9f41-4837-8662-a1d99118008d",
+  "itemsAndHoldingsFields": {
+    "items": [
+      {
+        "id": "0aaef1ea-69eb-4f6e-a077-cea159317ce3",
+        "volume": "volume",
+        "location": {
+          "location": {
+            "campusId": "470ff1dd-937a-4195-bf9e-06bcfcd135df",
+            "libraryId": "c2549bb4-19c7-4fcc-8b52-39e612fb7dbe",
+            "campusName": "Online",
+            "libraryName": "Online",
+            "institutionId": "40ee00ca-a518-4b49-be01-0638d0a4ac57",
+            "institutionName": "Københavns Universitet"
+          }
+        },
+        "callNumber": {
+          "prefix": "prefix",
+          "suffix": "suffix",
+          "typeId": "512173a7-bd09-490e-b773-17d83f2b63fe",
+          "callNumber": "TK5105.88815 . A58 2004 FT MEADE"
+        },
+        "enumeration": "enum",
+        "materialType": "sound recording",
+        "electronicAccess": [
+          {
+            "uri": "holding uri",
+            "name": "No information provided",
+            "linkText": "holding link text",
+            "publicNote": "holding public note",
+            "relationshipId": "f50c90c9-bae0-4add-9cd0-db9092dbc9dd",
+            "materialsSpecification": "holding mats"
+          },
+          {
+            "uri": "item uri",
+            "name": "Related resource",
+            "linkText": "item link text",
+            "publicNote": "item public note",
+            "relationshipId": "5bfe1b7b-f151-4501-8cfa-23b321d5cd1e",
+            "materialsSpecification": "item mast"
+          }
+        ]
+      }
+    ],
+    "instanceid": "1b74ab75-9f41-4837-8662-a1d99118008d"
```

Comment:
`instancedid` is a typo, should be `instanceId`.

Reply:
I didn't see this typo, possibly was fixed.

Reply:
It is still wrong. `id` starts with a lower i, it should be a capital I: `Id`
This still fails the build.

Reply:
I fixed to use camel-case wverywhere.